docker/Dockerfile

```diff
@@ -40,7 +42,7 @@ RUN python3 -m venv /venv && \
     /venv/bin/pip install --no-cache-dir --upgrade pip && \
     /venv/bin/pip install --no-cache-dir jupyter && \
     /venv/bin/pip install --no-cache-dir wheel && \
-    /venv/bin/pip install --no-cache-dir -e /app/devito[extras,mpi] && \
+    CFLAGS=-noswitcherror /venv/bin/pip install --no-cache-dir -e /app/devito[extras,mpi,nvidia] && \
```

Comment:
this is nvidia-specific, it shouldn't go here?

Reply:
I "think" it's would still install fine just wouldn't run if you tried Nvidia stuff